keeper/msg_server_test.go

```diff
@@ -1384,6 +1384,69 @@ func TestSwap(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestNegativeSwapResult(t *testing.T) {
+	bob := utils.TestAccount()
+	account := mocks.AccountKeeper{
+		Accounts: make(map[string]sdk.AccountI),
+	}
+	bank := mocks.BankKeeper{
+		Balances:    make(map[string]sdk.Coins),
+		Restriction: mocks.NoOpSendRestrictionFn,
+	}
+	bank.Balances[bob.Address] = append(bank.Balances[bob.Address], sdk.NewCoin("uusdc", math.NewInt(10000*ONE)))
+	bank.Balances[bob.Address] = append(bank.Balances[bob.Address], sdk.NewCoin("uusdn", math.NewInt(10000*ONE)))
+	k, ctx := mocks.SwapKeeperWithKeepers(t, account, bank)
+	server := keeper.NewMsgServer(k)
+	stableswapServer := keeper.NewStableSwapMsgServer(k)
+
+	// ARRANGE: Create a Pool.
```

Comment:
is there any significance to using `ARRANGE` in this comment? i'm mostly asking so i can understand if this is some convention we use for certain things.

Reply:
We prefer to write tests in the AAA style, which stands for **A**rrange, **A**ct, and **A**ssert.